server/core/src/main/java/org/infinispan/server/core/ServerManagement.java

```diff
@@ -12,7 +13,11 @@
 
    ConfigurationInfo getConfiguration();
 
-   void stop();
+   void serverStop();
```

Comment:
I would like to remove this method and only use `serverStop(List<String> servers)`.
if the list is null or empty, we shut down the current server.

Reply:
Ok